### PR TITLE
Update readme regarding maintenance

### PR DESCRIPTION
**Pull request by afshin**
This PR updates the readme with the strategy that we discussed in the weekly `jupyter/notebook` triage/maintenance meeting (08:30 Pacific Time on Wednesdays https://zoom.us/my/jovyan).

Once we agree on the language, we can use this to create a PR template as well.

**Please CC anyone you believe should see this.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ computing.
 
 ![Jupyter notebook example](docs/resources/running_code_med.png "Jupyter notebook example")
 
+Since the release of JupyterLab, many users of the classic Jupyter Notebook have been migrating to using the new front-end.
```
Comment (kevin-bates) [inline README.md:14]: This opening sentence feels a little out of place.  I wonder if it should be included in the paragraph below?  

FWIW, the pending 6.1 changelog currently uses this kind of language: 
>*Please note that this repository is currently maintained by a skeleton crew of maintainers from the Jupyter community. As a result, we encourage users to transition to JupyterLab, where more immediate support can occur.*

Or we could drop it allowing the `Alternatives to contributing...` paragraph to serve its purpose.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,19 @@ computing.
...
+
+The `jupyter/notebook` repository is currently maintained by a skeleton crew of maintainers from the Jupyter community. Our approach is:
```
Comment (kevin-bates) [inline README.md:16]: Suggestion:
Our approach is: -> As a result, our approach moving forward will be:

File: README.md [hunk 3]
```diff
@@ -11,6 +11,19 @@ computing.
...
+
+1. To maintain the security of the Jupyter Notebook. That means security-related issues and pull requests are our highest priority.
+2. To address JupyterLab [feature parity issues](https://github.com/jupyterlab/jupyterlab/issues?q=is%3Aopen+is%3Aissue+label%3A%22tag%3AFeature+Parity%22).
+3. To be responsive to the hard work of community members who have opened pull requests. We are triaging these PRs. We cannot support or maintain new features at this time, but we welcome security and other sustainability fixes.
+
+If you have an open pull request with a new feature or if you were planning to open one, please consider shipping it as a [notebook extension](https://jupyter-notebook.readthedocs.io/en/stable/extending/) instead.
+
+### Alternatives to contributing to `notebook`
```
[the remark this answers is not in the document]
Comment (Zsailer) [reply README.md:24]: I agree with Kevin's comment here. I thikn bold text would look best here.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zsailer) [reply README.md:14]: Unfortunately, I don't think you can make a *color box* in Github README files, but you can add emojis like ﻿:warning:

[the remark this answers is not in the document]
Comment (afshin) [reply README.md:15]: I think just having a header is the bare minimum. Good call, Kevin.

**Comment by Zsailer** [comment]
Thank you for doing this, @afshin!

**Comment by kevin-bates** [comment]
Nice catch Darian.  This looks good - thank you.

**Pull request merged**